includes/managers/widgets.php

```diff
@@ -374,6 +376,10 @@ public function ajax_get_widgets_default_value_translations( array $data = [] )
 				return ! empty( $widget['controls'] );
 			} )
 			->all();
+
+		$force_locale->restore();
```

Comment:
If you are able to force the language for a specific part, why do you need the AJAX?

Reply:
Because of the cache in `Controls_Stack::get_stack()` that will force us to add another parameter to the function, and it'll override the controls default values so we'll need to clean after that in order to prevent other `get_stack()`s from breaking.